### ISSUE TITLE
Two Y axis on same line graph

### ISSUE THREAD

**Issue by HarelM** [chinmaymk/angular-charts#193]
Hi,
Is it possible to show two Y axis on the same graph (one on the left and the other on the right for example)?
Thanks,

Harel M.

**Comment by bischofs** [comment]
I don't think so, this should be a decently high priority feature. It looks like Charts.js does not support it so that is probably why. I need this for a current project +1

**Comment by HarelM** [comment]
I've long switched to this library:
https://github.com/n3-charts/line-chart.
as this one seems abandoned...
